src/store/modules/services/services-mutations.ts

```diff
@@ -0,0 +1,31 @@
+import { MutationTree } from 'vuex'
+import { AvailableService, ServicesState } from '@/store/modules/services/types.ts'
+
+export const mutations: MutationTree<ServicesState> = {
+  useFastestService(state, value) {
+    state.useFastestService = value
+  },
+
+  toggle(state, payload: { type: AvailableService; url: string; active: boolean }) {
```

Comment:
I'd suggest to define types for payload on the top of file.

type TogglePayload = { type: AvailableService; url: string; active: boolean }